src/pages/TaskListPage/index.tsx

```
@@ -0,0 +1,128 @@
+    setPriorityFilter((prev: PriorityFilter) =>
+      prev === priority ? 0 : priority
+    );
+  };
+
+  const processTaskList = useMemo(() => {
+    const isValidDateString = (dateStr: string): boolean => {
+      const date = new Date(dateStr);
+      return !isNaN(date.getTime());
+    };
+
+    return [...taskList]
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)